### DISCUSSION TITLE
Fuzzy font on Windows

### DISCUSSION THREAD

**Discussion by simalei** [gtk-rs/gtk4-rs#1756] [Q&A]
Font is fuzzy for some reason on Windows. Enabling gtk-hint-font-metrics didn't help.

![image](https://github.com/gtk-rs/gtk4-rs/assets/65869620/9c11a503-13f9-41ec-95fc-46cc98725d16)

**Comment by bilelmoussaoui** [comment]
You are probably missing some dependency/configuration. But as I am not really using gtk on windows, I would recommend opening an issue upstream or using https://discourse.gnome.org/ to reach more gtk users that are not necessarily using the rust bindings.
